setup.py

```diff
@@ -45,7 +45,7 @@
         "pulpcore-client",
         "pyxdg",
         "requests",
-        "pulpcore-client",
+        "pulp-cli",
```

Comment:
I don't see, what this is trying to solve. Are any of the pulp-smash using tests depending on the cli?

Reply:
it is forcing click 8.0 and breaking pulp-cli tests